Formula/wxpython.rb

```diff
@@ -46,15 +54,14 @@ def install
     inreplace "wscript", "MACOSX_DEPLOYMENT_TARGET = \"10.6\"",
                          "MACOSX_DEPLOYMENT_TARGET = \"#{MacOS.version}\""
 
-    if OS.mac?
-      sdk = MacOS.sdk_path_if_needed
-      ENV.append_to_cflags "-I#{sdk}/usr/include" if sdk
+    if OS.mac? && (sdk = MacOS.sdk_path_if_needed)
```

Comment:
```suggestion
    if (sdk = MacOS.sdk_path_if_needed)
```
We don't need the `OS.mac?`. We might even be able to remove the parens.